cddl/conditional-series-record.cddl

```diff
@@ -1,6 +1,6 @@
 conditional-series-record = [
-  ; reference values to be matched against evidence
-  refv: measurement-values-map
+  ; reference values to be matched against evidence in each mkey
+  refv: [ + measurement-map ]
   ; endorsed values that apply in case revf matches
-  endv: measurement-values-map
+  endv: [ + mkeyvalue-pair ]
```

Comment:
```suggestion
  endv: [ + measurement-map]
```

Reply:
We don't want to have authorized-by in the endorsed values, since it is not meaningful here.

Reply:
It is not true, Endorsements or Reference Values, all should support Authority!

Reply:
No, authority in a measurement-map in a CoRIM represents a condition. Authority in a measurement-map in an ECT represents actual authorities that signed the information. For any _added_ measurement, the only authority is the CoRIM issuer. Given that the triple has already given two other positions for stating an authority condition, it doesn't make sense for the endv to have its own conditionality semantics.

Reply:
I understand your point, however, this has a deeper implications.

By not providing a source of Authority for Added Endorsements in the triple, has implications.

It means the supplier of Endorsements has to have an implicit authority of CoRIM Signer.

It cannot inherit the Authority from RefValue provider Authority has they may not be the same!

Which implies, that for a Env with ClassID : X, with FW running from Authority coming from FW Supplier PQR (in RefVal), the

Test House is vouching for its Endorsements.

It is fine to have CoRIM as a single source of Authority in this case, in most of the cases, so we can start with that.

I was thinking as Authority is `optional` retaining it will provide, the flexibility to introduce endorsements coming from different authority, For an Environement, like for FW Element -1 : Authority X, FW Element 2: Authority Y

Reply:
The only integrity protection you can ensure is who signed the information. https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/244 affirms that authorized-by is a condition, and not part of an addition. If it's not part of the condition and is actual addition, we need to have a triple signed by the delegate that the CoRIM issuer is authorized to sign on its behalf. I don't think we should do that though. You generally do that with PKI certificate paths.

Please express your example in terms of the conditional series triple so I may understand.